main.go

```diff
@@ -1,9 +1,31 @@
 package main
 
 import (
+	"dummy/controller"
+	"dummy/controller/api"
+	"dummy/database"
 	"dummy/handler"
+	"dummy/repository"
+	"log"
+	"net/http"
+	"os"
 )
 
 func main() {
-	handler.StartServer()
+	var err error
+	db, err := database.SetupDatabase()
+	if err != nil {
+		panic(err)
+	}
+	repo := repository.New(db)
+
+	// create controllers
+	homeController := controller.NewHome()
+	modelController := controller.NewModel(repo)
+	apiController := api.NewAPI(repo)
+
+	// route and start server
+	r := handler.HandleRouting(homeController, modelController, apiController)
+	port := os.Getenv("PORT")
```

Comment:
This is a little bit cheating, since the environment variables are loaded in `database.SetupDatabase()` above. I will refactor this in the next PR.

Reply:
Or perhaps you can do that too :-)